npm install failing on missing dependency

Here is what I see:

```
npm install --global standard-eject
npm ERR! Error while executing:
npm ERR! /usr/local/bin/git ls-remote -h -t https://github.com/parro-it/falafel-bash.git
npm ERR! 
npm ERR! remote: Repository not found.
npm ERR! fatal: repository 'https://github.com/parro-it/falafel-bash.git/' not found
npm ERR! 
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/salfuentes/.npm/_logs/2020-10-07T15_11_27_351Z-debug.log
```

npm --version
6.14.8
node --version
v14.10.0 


Please let me know if you'd like me to provide additional information.

--- a/package.json
+++ b/package.json
@@ -21,7 +21,6 @@
   },
   "homepage": "https://github.com/josephfrazier/standard-eject#readme",
   "dependencies": {
-    "falafel-bash": "https://github.com/parro-it/falafel-bash#656353237cc8f4deedf27b84ef617ef18072f3a6",
     "format-package-json": "^0.2.0",
     "has-yarn": "^1.0.0"
   },

--- a/package.json
+++ b/package.json
@@ -21,6 +21,7 @@
   },
   "homepage": "https://github.com/josephfrazier/standard-eject#readme",
   "dependencies": {
+    "falafel-bash": "^1.0.3",
     "format-package-json": "^0.2.0",
     "has-yarn": "^1.0.0"
   },

